src/sql/command.py

```diff
@@ -89,3 +87,55 @@ def connection(self):
     def result_var(self):
         """Returns the result_var"""
         return self.parsed["result_var"]
+
+    def _params_helper(self, params):
+        def is_float(element):
+            if element is None:
+                return False
+            try:
+                float(element)
+                return True
+            except ValueError:
+                return False
+
+        params_dict = {}
+        if not params:
+            return params_dict
+            # Throw some exception
+
+        for key, value in params:
+            if is_float(value):
+                params_dict[key] = int(value)
+            else:
+                params_dict[key] = value
+        return params_dict
+
+    def _var_expand(self, magic, user_ns, line, cell):
+        """
+        Support for the variable substition in the SQL clause
+        For now, we have enabled two ways:
+        1. Latest format, {{a}}, we use jinja2 to parse the string with {{a}} format
+        2. Legacy format, {a}, $a, and :a format.
+
+        We will deprecate the legacy format feature in next major version
+        """
+        self.is_legacy_var_expand_parsed = False
+        # Latest format parsing
+        # TODO: support --param and --use-global logic here
+        line = Template(line).render(user_ns)
+        cell = Template(cell).render(user_ns)
+        # Legacy format parsing
+        parsed_cell = magic.shell.var_expand(cell, depth=2)
+        parsed_line = magic.shell.var_expand(line, depth=2)
+        # Exclusive the string with "://", but has :variable
+        has_SQLAlchemy_var_expand = re.search("(?<!://):[^/]+", line) or ":" in cell
+
+        if parsed_line != line or parsed_cell != cell or has_SQLAlchemy_var_expand:
+            self.is_legacy_var_expand_parsed = True
+            print(
```

Comment:
let's change this for `warnings.warn` (FutureWarning) https://docs.python.org/3/library/exceptions.html#FutureWarning

Reply:
Add in [ff8c1d4](https://github.com/ploomber/jupysql/pull/137/commits/ff8c1d44bf03072dd547c49bec060c059c9c1377)

Reply:
@tonykploomber please resolve whatever you commit instead of leaving those open and outdated. It's cleaner.

Reply:
Cool, I thought I should let reviewer review again and let them close, will do next time